style-sample.css

```diff
@@ -0,0 +1,161 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.app-sample {    
+    height: 896px;
+    width: 1440px;
+    left: 0px;
+    top: 1680px;    
+    background-color: #D5FAFC;    
+}
+
+.section-heading {
+    height: 168px;
+    width: 445px;
+    top: 120px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    position: absolute;
+}
```

Comment:
As I can see all of the sections here are being absolutely positioned, it's not the proper way to do it. 
Absolute positioning should be relative to some other element. It's only applied when the normal positioning is not gonna work for it.

So change the layout of the sections and avoid using positioning. Just use flexbox and if necessary, margins and paddings.